src/stylesheets/components/_footer.scss

```diff
@@ -300,3 +300,50 @@ li.usa-footer-primary-content {
     padding-left: 1em;
   }
 }
+
+.usa-social_link {
+  // height of icon within hit area
+  $background-height: 3rem;
+
+  // Link hit target is 44 x 44 pixels following Apple iOS Human Interface Guidelines
+  $hit-area: 4.4rem; 
+
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: auto $background-height;
+  display: inline-block;
+  height: $hit-area;
+  left: 1.2rem; // relative left positioning
+  margin-left: 1rem; // spacing between link hit areas
+  position: relative;
+  text-align: center;
+  width: $hit-area;
+
+  span {
+    @include sr-only();
+  }
+}
+
+.usa-link-facebook {
+  @extend .usa-social_link;
+  background-image: url("../img/social-icons/png/facebook25.png");
+  background-image: url("../img/social-icons/svg/facebook25.svg");
+}
+
+.usa-link-twitter {
+  @extend .usa-social_link;
+  background-image: url("../img/social-icons/png/twitter16.png");
+  background-image: url("../img/social-icons/svg/twitter16.svg");
+}
+
+.usa-link-youtube {
+  @extend .usa-social_link;
+  background-image: url("../img/social-icons/png/youtube15.png");
+  background-image: url("../img/social-icons/svg/youtube15.svg");
+}
+
+.usa-link-rss {
+  @extend .usa-social_link;
+  background-image: url("../img/social-icons/png/rss25.png");
+  background-image: url("../img/social-icons/svg/rss25.svg");
+}
```

Comment:
Add one line-break here. All of our files should have an extra space at the end of the file.